sites/footers/index.json-ld

```diff
@@ -29,6 +29,16 @@
 				"language": "fr",
 				"path": "footers-fr.html"
 			},
+			{
+				"title": "Complete footer (deprecated)",
+				"language": "en",
+				"path": "footers-en.html"
+			},
+			{
+				"title": "Pied de page complet (d�conseill�)",
```

Comment:
There is an UTF-8 encoding issue here.

Reply:
Aww lame lol.

Since this isn't the first time ANSI has messed up one of my GCWeb PRs, I decided to check out what was going on...

Turns out that even though GCWeb's .editorconfig file sets the default charset to UTF-8, Notepad++'s editorconfig plugin doesn't currently support that property (see https://github.com/editorconfig/editorconfig-notepad-plus-plus/issues/41#issuecomment-959124854). That leaves it up to Notepad++ to auto-detect file encoding. Since this file didn't previously contain any French characters in it, Notepad++ auto-detected it as ANSI when I opened it for editing. So it saved my accented characters as ANSI.

GitHub's diff viewer probably correctly handles the .editorconfig file's charset property and consequently showed my ANSI characters as messed-up.